### DISCUSSION TITLE
Release notes

### DISCUSSION THREAD

**Discussion by rogerthn2019** [UtilitechAS/amsreader-firmware#687] [Ideas]
First, THANKS for a great project!

It would be useful, for myself if changes in data.json format was included in the release notes.
Not critical but it would have saved me some time and headache ;-)

**Comment by Noschvie** [comment]
Which changes do you mean?

**Comment by rogerthn2019** [comment]
Addition of i2e after i2 meaning that "index" of all following is increased by 1.

**Comment by gskjold** [comment]
Thanks for you support! I have updated the wiki. I do not announce additions to the json files. These are expected to be read with a json parser, in which case there would be no adverse effect of adding fields.
